content/en/docs/zero-code/java/quarkus/getting-started.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Getting started with Quarkus
+weight: 20
+---
+
+To enable OpenTelemetry in your Quarkus application, add the
+`quarkus-opentelemetry` extension dependency to your project.
+
+{{< tabpane >}} {{< tab header="Maven (`pom.xml`)" lang=Maven >}}
+
+````xml
+
+{{< tabpane text=true >}} {{% tab header="Maven (`pom.xml`)" lang=Maven %}}
+
+```xml
+<dependency>
+    <groupId>io.quarkus</groupId>
+    <artifactId>quarkus-opentelemetry</artifactId>
+</dependency>
+````
```

Comment:
~~~suggestion
```
~~~

Reply:
Strange... I did run `npm run fix:all` on this [commit](https://github.com/open-telemetry/opentelemetry.io/pull/6138/commits/459409f3121a4e741ca8dc04a0dd0632206301d7) and it seems that it created these errors.